### PR TITLE
LPS-127324 Pass correct function to openFormViewModal on step form view

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Review by albinojunior — APPROVED** [review]

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js [hunk 1]
```diff
@@ -218,9 +216,7 @@ export default function DataAndViewsTab({
 							({id}) => id === newFormView.id
 						);
 
-						if (
-							!(currFormView.nativeField && currFormView.missing)
-						) {
+						if (!currFormView.missingRequiredFields?.nativeField) {
```
Comment (rafaprax) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js:219]: Hey @naraandrade Can we rename this to currentFormView?

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**